### PR TITLE
Added Echis Household Reports and Data sources

### PR DESCRIPTION
**Pull request by echisethiopia**

### CODE REVIEW

**Comment by echisethiopia** [comment]
@sravfeyn  Here is another pull req, I added the changed files and the tests passes too.

**Comment by sravfeyn** [comment]
@echisethiopia Thanks.

Can you remove below two commits that are modifying our core code. I have also commented on the file diffs in this PR.
- https://github.com/dimagi/commcare-hq/commit/f5da7243ffd0f3862a2f53558534bee26d401f41
- https://github.com/dimagi/commcare-hq/commit/65dd29f8d6f8f742ae894fd1ee9ef0e3f646a4de

File: COMMON_ISSUES.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Common issues
 
 + If you have an authentication error running `./manage.py migrate` the first
   time, open `pg_hba.conf` (`/etc/postgresql/9.1/main/pg_hba.conf` on Ubuntu)
-  and change the line "local all all peer" to "local all all md5".
+  and change the line "local all all peer" to "local all all md5".
```
Comment (sravfeyn) [inline COMMON_ISSUES.md]: Can you undo this please?

File: custom/apps/wisepill/models.py [hunk 2]
```diff
@@ -6,7 +6,6 @@
 
 from corehq.util.python_compatibility import soft_assert_type_text
 
-
 class WisePillDeviceEvent(Document):
```
Comment (sravfeyn) [inline custom/apps/wisepill/models.py]: Can you undo this please?

**Comment by echisethiopia** [comment]
@sravfeyn  I reverted back my initial changes.

**Comment by echisethiopia** [comment]
test failed.  updated the fork and there are around 47 changes.  will merge those and push.

**Comment by sravfeyn** [comment]
Thanks.

**Comment by echisethiopia** [comment]
@sravfeyn , I merged and pushed , the test failed.
[this ](https://help.github.com/en/articles/syncing-a-fork)is how I merged and pushed.

**Comment by echisethiopia** [comment]
humm, I will create another fork again and push only the changes.

**Comment by echisethiopia** [comment]
@sravfeyn  now it passed

**Comment by sravfeyn** [comment]
Thank you for all the patience in getting this PR merged!

**Comment by echisethiopia** [comment]
Thank you for all your help

**Pull request merged**